code/menuui/playermenu.cpp

```diff
@@ -154,8 +154,8 @@ int Player_select_num_pilots;						// # of pilots on the list
 int Player_select_list_start;						// index of first list item to start displaying in the box
 int Player_select_pilot;							// index into the Pilot array of which is selected as the active pilot
 int Player_select_input_mode;						// 0 if the player _isn't_ typing a callsign, 1 if he is
-char Pilots_arr[MAX_PILOTS][MAX_FILENAME_LEN];
-char *Pilots[MAX_PILOTS];
+char Pilots_arr[MAX_PILOTS][MAX_FILENAME_LEN];		// Filename of associated Pilot. Same idx as Pilots[]
+char *Pilots[MAX_PILOTS];							// Pilot callsigns
```

Comment:
Also can this be made into a `const char *`?

Reply:
Which one? `Pilots_arr[][]` or `Pilots[]`?

Reply:
`Pilots[]`

Reply:
No, `Pilots[]` can not be `const char *` because the menu code modifies the array to do sorting and adding new pilots.  It may be possible to do this change but it would require more extensive changes elsewhere.